libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -342,6 +342,12 @@ private object EvalExpression(string exp)
 
             if (result == null)
             {
+                if (tolerant)
+                {
+                    // temporary solution
```

Comment:
it might not be a temporary solution, so my suggestions is remove this comment because it might be very mis-leading without detail plans